cmd/brew-test-bot.rb

```diff
@@ -1090,6 +1090,11 @@ def homebrew
         test "brew", "man", "--fail-if-changed"
         test "brew", "style"
       elsif @tap
+        installed_taps = Tap.select(&:installed?).map(&:name)
```

Comment:
I think this should be done only when `@test_brew` isn't set (rather than needing a `@tap`) and might make more sense in `def download`, thoughts?

Reply:
It sure might be a better way of doing that. Just force-pushed this change.